src/ApiService/ApiService/QueueFileChanges.cs

```diff
@@ -52,11 +52,11 @@ public QueueFileChanges(ILoggerFactory loggerFactory, IStorageProvider storagePr
             return Task.CompletedTask;
         }
 
-        file_added(fileChangeEvent, lastTry);
+        file_added(log, fileChangeEvent, lastTry);
```

Comment:
This pattern of passing the `log` around is only temporary until we move `var log = _loggerFactory.MakeLogTracer(Guid.NewGuid());` into middleware right?

Reply:
yes, that was my thinking.

Correlation ID might come from CLI as well (we should allow that at least), or from a message on a Queue, since we are tracking message progress through the system